lib/jormungandr/src/Cardano/Wallet/Jormungandr/Api/Client.hs

```diff
@@ -143,7 +148,8 @@ data JormungandrClient m = JormungandrClient
         :: Hash "Genesis"
         -> ExceptT ErrGetBlockchainParams m (J.Block, BlockchainParameters)
     , getStakeDistribution
-        :: ExceptT ErrNetworkUnavailable m StakeApiResponse
+        :: ExceptT ErrNetworkUnavailable m
+            (EpochNo, [(PoolId, Quantity "lovelace" Word64)])
```

Comment:
Changed the type here in `JormungandrClient`. We don't need the full `StakeApiResponse`.

Reply:
Hmmmm. But now our client is not really a client :/ ... Please reverse to `StakeApiResponse` here :+1: